aiidalab_widgets_base/wizard.py

```diff
@@ -142,11 +142,13 @@ def __init__(self, steps, **kwargs):
         )
         self.next_button.on_click(self._on_click_next_button)
 
-        header = ipw.HBox(
+        self.header = ipw.HBox(
             children=[self.back_button, self.reset_button, self.next_button]
         )
 
-        super().__init__(children=[header, self.accordion], **kwargs)
+        if hide_header:
+            self.header.layout.display = "none"
+        super().__init__(children=[self.header, self.accordion], **kwargs)
```

Comment:
Should we add a method to display or hide it as well ?

Reply:
Thanks for the suggestion. I updated the code, now the user can set
```python
# switch on the header
app.show_header=True
# switch off 
app.show_header=False
```